src/state/types.ts

```diff
@@ -23,11 +24,12 @@ export interface PlayerState {
   deck: Card[];
   hand: Card[];
   discard: Card[];
-  ability: AbilityTile[];
+  abilities: AbilityTile[];
 }
 
 export interface AbilityTile extends GamePieceBase {
   type: "ability";
+  is_used: boolean;
```

Comment:
Any specific reason for snake_case here?

Reply:
hmm. nope =) Lets stick to camelCase?